### ISSUE TITLE
error trying to enable bb on b4.0

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#5012]
error trying to enable bb on b4.0

[blenderbim-240712: Update shape_builder.py](https://github.com/IfcOpenShell/IfcOpenShell/releases/tag/blenderbim-240712)
```
os: Windows
os_version: 10.0.19045
python_version: 3.10.13
architecture: ('64bit', 'WindowsPE')
machine: AMD64
processor: Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
blender_version: 4.0.2
blenderbim_version: 0.0.240712
last_actions: 
last_error: Traceback (most recent call last):
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\__init__.py", line 96, in <module>
    import blenderbim.bim
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\bim\__init__.py", line 25, in <module>
    from . import handler, ui, prop, operator, helper
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\bim\handler.py", line 25, in <module>
    import blenderbim.tool as tool
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\tool\__init__.py", line 30, in <module>
    from blenderbim.tool.document import Document
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\tool\document.py", line 23, in <module>
    from blenderbim.bim import import_ifc
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\bim\import_ifc.py", line 44, in <module>
    from blenderbim.tool.loader import OBJECT_DATA_TYPE
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\tool\loader.py", line 42, in <module>
    class Loader(blenderbim.core.tool.Loader):
  File "C:\Users\Ryan Schultz\AppData\Roaming\Blender Foundation\Blender\4.0\scripts\addons\blenderbim\tool\loader.py", line 59, in Loader
    def get_mesh_name(cls, geometry: ifcopenshell.geom.ShapeType) -> str:
AttributeError: module 'ifcopenshell.geom' has no attribute 'ShapeType'
```

**Comment by Moult** [comment]
Is this a dev setup or a fresh install?

**Comment by theoryshaw** [comment]
fresh :)

**Comment by Moult** [comment]
Oh, I see now. Basically two days ago I merged a PR into 0.7.0 (I didn't notice it was merging to the wrong location) and so it ended up doing a release from the wrong branch with old (broken) code. So you just happened to choose a bad day :) This shouldn't happen in the future I hope.
